projects/plugins/jetpack/modules/stats.php

```diff
@@ -897,43 +898,44 @@ function stats_admin_bar_head() {
 }
 
 /**
- * Stats AdminBar.
+ * Gets the image source of the given stats chart.
  *
- * @access public
- * @param mixed $wp_admin_bar WPAdminBar.
- * @return void
+ * @param string $chart Name of the chart.
+ * @param array  $args Extra list of argument to use in the image source.
+ * @return string An image source.
  */
-function stats_admin_bar_menu( &$wp_admin_bar ) {
-	$url = add_query_arg( 'page', 'stats', admin_url( 'admin.php' ) ); // no menu_page_url() blog-side.
+function stats_get_image_chart_src( $chart, $args = array() ) {
+	$url = add_query_arg( 'page', 'stats', admin_url( 'admin.php' ) );
 
-	$img_src    = esc_attr(
-		add_query_arg(
-			array(
-				'noheader' => '',
-				'proxy'    => '',
-				'chart'    => 'admin-bar-hours-scale',
-			),
-			$url
-		)
-	);
-	$img_src_2x = esc_attr(
-		add_query_arg(
+	return add_query_arg(
+		array_merge(
 			array(
 				'noheader' => '',
 				'proxy'    => '',
-				'chart'    => 'admin-bar-hours-scale-2x',
+				'chart'    => $chart,
 			),
-			$url
-		)
+			$args
+		),
+		$url
 	);
+}
 
-	$alt = esc_attr( __( 'Stats', 'jetpack' ) );
-
-	$title = esc_attr( __( 'Views over 48 hours. Click for more Site Stats.', 'jetpack' ) );
+/**
+ * Stats AdminBar.
+ *
+ * @access public
+ * @param mixed $wp_admin_bar WPAdminBar.
+ * @return void
+ */
+function stats_admin_bar_menu( &$wp_admin_bar ) {
```

Comment:
Nitpick: Can you add the `public` visibility keyword.

Reply:
https://github.com/Automattic/jetpack/pull/21655#discussion_r745486060